components/search_results_item/index.js

```diff
@@ -23,7 +25,23 @@ import {getDirectTeammate, getDisplayNameByUser} from 'utils/utils.jsx';
 
 import SearchResultsItem from './search_results_item.jsx';
 
+export function makeGetReplyCount() {
+    return createSelector(
+        (state) => state.entities.posts.posts,
+        (state, post) => state.entities.posts.postsInThread[post.root_id || post.id],
+        (allPosts, postIds) => {
+            if (!postIds) {
+                return 0;
+            }
+
+            // Count the number of non-ephemeral posts in the thread
+            return postIds.map((id) => allPosts[id]).filter((post) => post && !isPostEphemeral(post)).length;
+        }
+    );
+}
+
```

Comment:
This is the same as in [Post component](https://github.com/mattermost/mattermost-webapp/blob/6c0a915972b8d671db5676b6ec04554911c334a5/components/post_view/post/index.js#L40-L53).  How about move the function and corresponding tests to `utils/post_utils.jsx` and import from there?